app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>Here is the details of a given user</h1>
+<section class = "user-profile">
+    <section class = "user-card">
+      <div class="user-img">
+        <img src="<%= @users.photo %>" class="user-photo">
+      </div>
+      <div class="user-info">
+        <h1><%= @users.name %></h1>
+        <h2 class="post-counter"> Number of posts: <%= @users.posts_counter%></h2>
+      </div>
+    </section>
+    
+    <section class = "user-bio">
+      <h5>Bio</h5>
+      <%= @users.bio %>
+    </section>
+    <section class = "user-posts">
+      <% @users.recent_posts.each do |post| %>
+        <div class= "post">
+          <h2><%= post.title %></h2>
+          <p><%= post.text %></p>
+          <div>
+            <p>Comments: <%= post.comments_counter %></p>
+            <p>Likes: <%= post.likes_counter %></p>
+        </div>
```

Comment:
- [ Optional ] I think it would be nice if you `style` the `recent_posts` section the same way it has been styled in the project wireframe. Currently, it appears like the comments section varies from the project wireframe in the following ways, please see the screenshot below and rectify this.

  - [ ] The `comments` and `likes` text in your design are aligned to the left while in the project wireframe, they are aligned to the right-hand side.
  - [ ] The `comments` box in your design does not have a border while in the project wireframe it has one.

## Your design 
![Screenshot from 2023-11-04 01-53-30](https://github.com/Wineshuga/pen-it/assets/16392046/6f95a39b-8264-43ef-b789-e0e4fdbbe6ed)

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABTYBUDYLDWA5YD6WCUH3UIYZKFRE4A)